samples/src/PopupPluginSample/Views/FLPRoot.xaml.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Prism.Navigation;
+using Xamarin.Forms;
+
+namespace PopupPluginSample.Views
+{
+    public partial class FLPRoot : FlyoutPage, IMasterDetailPageOptions
```

Comment:
this should use IFlyoutPageOptions

Reply:
[fixed](https://github.com/dansiegel/Prism.Plugin.Popups/pull/226/commits/bcd833996395875f9036264e927bb914a66f46ea)